docs/querying.md

```diff
@@ -467,6 +467,17 @@ Subtask.findAll({
 })
 ```
 
+`order` is also supported in the `destroy` method, but will only accept an an array of attributes and directions.
```

Comment:
Why can't we use order processing that other methods support, any specific reason. We can possibly use helpers that are used to build order statement for findAll etc